src/multibody/joint/joint-base.hpp

```diff
@@ -356,6 +357,15 @@ namespace se3
      */
     void normalize_impl(Eigen::VectorXd &) const { }
 
+    /**
+     * @brief      Check if the DoFs of the joint are equivalents for two configurations
```

Comment:
It should be rather: `Check if two configurations are equivalent within the given precision`